buildSrc/src/main/java/io/reactor/gradle/JavaConventions.java

```diff
@@ -59,6 +59,8 @@ private void applyJavaConvention(Project project) {
 					       "-Xlint:processing",
 					       "-Xlint:static",
 					       "-Xlint:try",
+						   "-Xmaxerrs", "1000",
```

Comment:
Could you please clarify this change?

Reply:
I can since I did a similarly large change in the past (but abandoned it): javac will only show 100 errors by default. it is a bit cumbersome and quite disheartening to fix these 100 errors only to find 100 new ones have appeared (because there is actually a total of eg. 384 errors in the codebase)

Reply:
yes, this is something that helped me grasp the total amount or errors I needed to fix (and where). May not need to be left in the final merge.